index.html

```diff
@@ -10,74 +10,112 @@
-        <li>CSS</li>
-        <li>JavaScript</li>
+      <ul class="no-bullets">
+        <li class="buttontext inline">HTML</li>
+        <li class="buttontext inline">CSS</li>
+        <li class="buttontext inline">JavaScript</li>
       </ul>
-    </p>
+    </div>
   </section>
+  </summary>
 
   <hr />
 
-  <section id="contact">
-    <h2>Contact</h2>
-    <div class="centre">Please feel free to contact me if you have any questions.
-      <br>
-      Below you can find my contact details:
-    
-      <address>
-       <a href="mailto:jelena_de_la_sauce@web.de" class="buttontext">jelena_de_la_sauce@web.de</a>
-       <br>
-       <a href="tel:+4915159092870" class="buttontext">+49 151 59092870</a>
-      </address>
-    </div>
-
-    <br>
+  <h2 id="contact">Contact</h2>
+  <div>Please feel free to contact me if you have any questions.</div>
+  <section class="container-contact">
+      <address class="child-contactadr">
+        <ul class="no-bullets container-address">
+          <li class="inline"><a href="mailto:jelena_de_la_sauce@web.de">jelena_de_la_sauce@web.de</a></li>
+          <li class="inline"><a href="tel:+4915159092870">+49 151 59092870</a></li>
+      </ul>
+    </address>
 
-    <div class="centre"> You can also folow me on:
-    <br>
-      <a href="https://www.linkedin.com/in/jelena-de-la-sauce-33653577/" target="_blank"> <img src="https://upload.wikimedia.org/wikipedia/commons/c/c2/F_icon.svg" alt="" width="50" height="50" class="contact"></a>
-      <a href="https://github.com/JelenadlS" target="_blank"> <img src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" alt=""width="50" height="50" class="contact"></a>
+    <div class="child-contactmed"> 
+      <ul class="no-bullets container-media">
+        <li class="inline"><a href="https://www.linkedin.com/in/jelena-de-la-sauce-33653577/" target="_blank"> <img src="https://upload.wikimedia.org/wikipedia/commons/c/c2/F_icon.svg" alt="" width="200" height="200" class="contact"></a></li>
+        <li class="inline"><a href="https://github.com/JelenadlS" target="_blank"> <img src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" alt="" width="200" height="200" class="contact"></a></li>
+        <li class="inline"><a href="https://www.linkedin.com/in/jelena-de-la-sauce-33653577/" target="_blank"> <img src="https://upload.wikimedia.org/wikipedia/commons/0/01/LinkedIn_Logo.svg" alt="" width="300" height="300" class="contact"></a></li>
+      </ul>
     </div>
   </section>
 
   <hr />
+<div class="abio">
+  <section id="about" class="abio-ab">
+    <h2>About</h2>
+    <p>I am a well organized and communicative Web Development student, who is graduating the boothcamp at neuefische end of March 2022. I am higly motivated to join a young, creative and inspiring team.</p>
+  </section>
+<hr />
 
-
-  <section id="biography">
+  <section id="biography" class="abio-bio">
     <h2>Biography</h2>
+    <table>
```

Comment:
Compliments for building up that tables in html 😃

Reply:
haha thank you, it is actually quite easy to build it in HTML - lets see how easy it will be to make it nice later on 😅